package.json

```diff
@@ -111,6 +111,7 @@
     "eslint/**/glob-parent": "^5.1.2",
     "eslint/**/ansi-regex": "^5",
     "prettier-eslint-cli/**/ansi-regex": "^5",
-    "isomorphic-fetch/**/node-fetch": "^2.6.7"
+    "isomorphic-fetch/**/node-fetch": "^2.6.7",
+    "browserify-sign": "4.2.2"
```

Comment:
I checked and `yarn-audit-fix` is able to resolve this without resorting to resolutions which is unnecessarily fragile